eng/packages/http-client-csharp-mgmt/generator/TestProjects/Local/Mgmt-TypeSpec/src/Generated/PrivateLinkResourceData.cs

```diff
@@ -27,6 +27,9 @@ internal PrivateLinkResourceData(ResourceIdentifier id, string name, string @typ
         /// <summary> The resource-specific properties for this resource. </summary>
         public PrivateLinkResourceProperties Properties { get; }
 
+        /// <summary> The name of the private link associated with the Azure resource. </summary>
+        public new string Name => _name ?? default;
```

Comment:
Is this expected?

Reply:
did not regen with the latest, will regen after applying the fix in https://github.com/Azure/azure-sdk-for-net/pull/49499#discussion_r2055211204

Reply:
Actually, I just want to make the generated code compile in this PR.
Because we are going to apply type replacement, so that [Resource.cs](https://github.com/Azure/azure-sdk-for-net/blob/d39265eeddb05d26808519daf9d490deaff8e73e/eng/packages/http-client-csharp-mgmt/generator/TestProjects/Local/Mgmt-TypeSpec/src/Generated/Models/Resource.cs#L15) will not be generated anymore.
We will handle this properly after https://github.com/Azure/azure-sdk-for-net/pull/49098